src/univariate/continuous/normal.jl

```diff
@@ -131,14 +131,14 @@ function suffstats(::Type{<:Normal}, x::AbstractArray{T}) where T<:Real
     n = length(x)
 
     # compute s
-    s = 0
+    s = zero(T)
```

Comment:
I guess even safer would be
```suggestion
    s = zero(T) + zero(T)
```
(e.g. if `T = Bool`) but usually it should not matter in practice I assume.